list.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'enumerable'
+
+class List
+  include Enumerable
```

Comment:
- [ ] Please you project is not running because you imported  `Enumerable` instead of `Enumerator` as it is spelled in `enumerable.rb`,  it is spelled as `Enumerator`, please kindly fix the issue and ensure that your project is running properly.

Reply:
Thank you, I have fixed the issue